README.md

```diff
@@ -23,71 +26,81 @@ Also, check the documentation of Wrangler Transform [here](transform/docs/Wrangl
 * Recipes
   * [Log Parsing](demos/Parsing-Apache-Log.md)
   * [HL7 CCDA XML Parsing](demos/HL7-CCDA-XML-Parsing.md)
-  * [CSV Parsing and micro-wrangling columns](demos/CSV-Parsing-And-Extraction.md)
+  * [CSV Parsing and Extracting Column Values](demos/CSV-Parsing-And-Extraction.md)
+
 
-  
 ## Concepts
 
-This implementation of wrangler defines the following concepts. Please familiarize yourself with these concepts. 
+This implementation of Wrangler uses these concepts.
 
 ### Record
 
-A Record is a collection of field names and field values. 
+A *Record* is a collection of field names and field values.
 
 ### Column
 
-A Column is a data value of any supported java type, one for each Record.
+A *Column* is a data value of any of the supported Java types, one for each record.
 
 ### Directive
 
-A Directive is a single data manipulation instruction specified to either transform, filter or pivot a single record into zero or more records. A directive can generate one or more Steps to be executed by the Pipeline. 
+A *Directive* is a single data manipulation instruction, specified to either transform,
+filter, or pivot a single record into zero or more records. A directive can generate one
+or more *steps* to be executed by a Pipeline.
 
 ### Step
 
-A Step is a implementation of a data transformation function operating on a Record or set of records. A step can generate zero or more Records from the application of a function. 
+A *Step* is an implementation of a data transformation function, operating on a single
+record or set of records. A step can generate zero or more records from the application of
+a function.
 
 ### Pipeline
 
-A Pipeline is a collection of Steps to be applied on a Record. Record(s) outputed from each Step is passed to the next Step in the pipeline. 
+A *Pipeline* is a collection of steps to be applied on a record. The record(s) outputed
+from a step are passed to the next step in the pipeline.
 
 ## Notations
 
 ### Directives
 
-A directive is represented as simple text in the format as specified below
+A directive is represented by simple text in this format:
```

Comment:
This is technically a TextDirective. For example, If a user wants to create a JSONDirective, it could be specified in the format { "action":"something","field":"else"}